include/kernels/dslash_mdw_fused.cuh

```diff
@@ -0,0 +1,451 @@
+#include <gauge_field_order.h>
+#if (CUDA_VERSION >= 10010 && __COMPUTE_CAPABILITY__ >= 700)
```

Comment:
Since this code is _very_ NVIDIA GPU specific, should we put this file in a target-specific directory?

Reply:
Let's leave this as is for now and address this is in a future PR.  At the moment all the functors live in `include/kernels` directory.  I'm thinking a better approach might be to move them all to `include/targets/generic/functors` directory (renaming `kernels` to `functors` at the same time).  Then we could easily add target-specialized functors to, for example the `include/targets/cuda/functors` directory.  For this specific kernel, we would move it to the CUDA directory, and leave a stub functor in the generic directory, allowing anyone to implement a generic variant if they want.